raiden/tests/unit/test_transport.py

```diff
@@ -11,11 +13,25 @@
 slogging.configure(':DEBUG')
 
 
+class IntervalCheck(object):
```

Comment:
Will add the docstring. I used the `IntervalCheck` name because I'm just asserting/checking the elapsed time.